### PR TITLE
[WIP] Remove data files from dash-tsne

### PR DESCRIPTION
**Pull request by xhlulu**
Issue for app: #[issue number here]

# App pull request
- [ ] This is a new app
- [ ] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version):
- Current gallery app URL: (delete this line if inapplicable)
- Python app repository link: (delete this line if you are working on a Python app)

## Workflow

- [ ] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [ ] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [ ] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [ ] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [ ] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [ ] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,6 @@ venv/
 apps/dash-loan-grade/.ipynb_checkpoints/Train Decision Trees-checkpoint.ipynb
 .ipynb_checkpoints/*
 apps/*/.ipynb_checkpoints/*
-apps/*/__pycache__/*
\ No newline at end of file
+apps/*/__pycache__/*
+apps/dash-tsne/data
+apps/dash-tsne/demo_embeddings
```
Comment (gzork) [inline .gitignore:14]: I'd add a [newline](https://unix.stackexchange.com/a/18789) here.

File: apps/dash-tsne/Procfile [hunk 2]
```diff
@@ -1 +1 @@
-web: ./launch.sh
\ No newline at end of file
+web: bash ./launch.sh
```
Comment (gzork) [inline apps/dash-tsne/Procfile:2]: I'd add a [newline](https://unix.stackexchange.com/a/18789) here too.

File: apps/dash-tsne/launch.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+wget https://github.com/plotly/datasets/releases/download/v0.0.1-data/dash-tsne.zip
+unzip dash-tsne.zip
+gunicorn app:server --workers 4
```
Comment (gzork) [inline apps/dash-tsne/launch.sh:3]: I'd add a [newline](https://unix.stackexchange.com/a/18789) here as well.

**Comment by xhlulu** [comment]
Thank you!

**Pull request merged**